_posts/2020-11-01-Fortran-Newsletter-November-2020.md

```diff
@@ -0,0 +1,190 @@
+---
+
+Welcome to the November 2020 edition of the monthly Fortran newsletter.
+The newsletter comes out on the first calendar day of every month and details
+Fortran news from the previous month.
+
+<ul id="page-nav"></ul>
+
+## fortran-lang.org
+
+This month we've had a few additions and improvements to the website:
+
+* [#152](https://github.com/fortran-lang/fortran-lang.org/pull/152):
+New mini-book on setting up the Fortran development environment.
+You can read it [here](https://fortran-lang.org/learn/os_setup).
+* [#147](https://github.com/fortran-lang/fortran-lang.org/pull/147):
+Automated posting to @fortranlang Twitter using twitter-together.
+* [#155](https://github.com/fortran-lang/fortran-lang.org/pull/155):
+Fix for a security vulnerability reported by the GitHub Security Team.
+* The following packages were added to the
+[Package Index](https://fortran-lang.org/packages):
+atomsk, ddPCM, DFTB+, DFT-D4, ELPA, ELSI, FortJSON, fypp, HANDE, libmbd, libnegf,
+mpifx, NTPoly, NWChem, OpenMolcas, PoisFFT, QMD-PROGRESS, scalapackfx,
+tapenade, wannier90, and xtb.
+* [#145](https://github.com/fortran-lang/fortran-lang.org/pull/145),
+[#146](https://github.com/fortran-lang/fortran-lang.org/pull/146),
+[#154](https://github.com/fortran-lang/fortran-lang.org/pull/154),
+[#158](https://github.com/fortran-lang/fortran-lang.org/pull/158):
+Minor fixes and improvements.
+
+Ongoing work:
+
+* [#160](https://github.com/fortran-lang/fortran-lang.org/pull/160) (WIP):
+In-depth introduction for Fortran with Make.
+* [#156](https://github.com/fortran-lang/fortran-lang.org/pull/156) (WIP):
+Updating the mini-book on building programs.
+
+[Let us know](https://github.com/fortran-lang/fortran-lang.org/issues) if you have any suggestions for the website and its content.
+We welcome any new contributors to the website and the tutorials page in particular - see the [contributor guide](https://github.com/fortran-lang/fortran-lang.org/blob/master/CONTRIBUTING.md) for how to get started.
+
+## Fortran Standard Library
+
+This month progress was made on a few pull requests:
+
+* [#239](https://github.com/fortran-lang/stdlib/pull/239): Implementation of the `stdlib_bitsets` module. It provides a bitset data type.
+* [#240](https://github.com/fortran-lang/stdlib/pull/240): Implementation of the `stdlib_stats_distribution` module. It provides probability distribution and statistical functions.
+* [243](https://github.com/fortran-lang/stdlib/pull/243): A proposition to support newline characters in the message provided to the logger.
+
+Don't hesitate to test and review these pull requests!
+
+Otherwise, ongoing discussions continue;
+ * [220](https://github.com/fortran-lang/stdlib/pull/220): API for file system operations: directory manipulation
+  * [241](https://github.com/fortran-lang/stdlib/pull/241): Include a `split` function (202X feature)
```

Comment:
Add hashes to a few of the PR numbers:

```suggestion
* [#243](https://github.com/fortran-lang/stdlib/pull/243): A proposition to support newline characters in the message provided to the logger.

Don't hesitate to test and review these pull requests!

Otherwise, ongoing discussions continue;
 * [#220](https://github.com/fortran-lang/stdlib/pull/220): API for file system operations: directory manipulation
 * [#241](https://github.com/fortran-lang/stdlib/pull/241): Include a `split` function (202X feature)
```